assets/activation-scripts/activate

```diff
@@ -163,9 +163,14 @@ if [ $flox_env_found -eq 0 ]; then
     echo >&2
   fi
 
-  # First activation of this environment. Snapshot environment to start.
-  _start_env="$($_coreutils/bin/mktemp --suffix=.start-env)"
-  export | $_coreutils/bin/sort > "$_start_env"
+  if [ $flox_env_found -eq 0 ]; then
+    # First activation of this environment. Snapshot environment to start.
+    _start_env="$($_coreutils/bin/mktemp --suffix=.start-env)"
+    export | $_coreutils/bin/sort > "$_start_env"
+  else # we know "$_FLOX_ACTIVATE_FORCE_REACTIVATE" == true
+    # TODO: restore _start_env
+    :
```

Comment:
We rely on the existence of `$start_env` below so we have to put _something_ here.

We can either keep `_{start,end}_env` around in the environment in which case we just affirm here that it is defined (and exists), or we can take this opportunity to think about how we would use the registry to track the env files for this activation.